great_expectations/expectations/core/expect_column_max_to_be_between.py

```diff
@@ -132,6 +132,7 @@ class ExpectColumnMaxToBeBetween(ColumnExpectation):
             bootstrap_random_seed=f"{VARIABLES_KEY}bootstrap_random_seed",
             round_decimals=f"{VARIABLES_KEY}round_decimals",
             truncate_values=f"{VARIABLES_KEY}truncate_values",
+            include_batch_id_with_metric_value=False,
```

Comment:
I would rather _remove_ this option and _always_ include batch ID when we can. It simplifies the API and code

Reply:
This has been implemented.